### PR TITLE
Documentation reviewed and updated

### PR DESCRIPTION
**Pull request by marikrg**
### Description
Update the CHANGELOG and README documentation

### Issues Resolved
No issue.

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

**Review by fgbulsoni — CHANGES_REQUESTED** [review]
All changes look good.
One more suggestion:
In the resources.md, instead of having Oneview summary -> all oneview resources -> i3s summary -> all i3s resources,
how about we make it so that the summary is actually the reference to the whole document?
I'd suggest having the i3s links up there, right bellow the oneview links, for ease of searching and finding resources.

**Comment by marikrg** [comment]
@fgbulsoni 

:memo: done!
Can you check it again?

File: RESOURCES.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Resources for the Puppet Module for HPE OneView
+# Summary
+
+## HPE OneView resources
```
Comment (fgbulsoni) [inline RESOURCES.md:3]: This should also be linking to the section where the oneview resources begin

File: RESOURCES.md [hunk 2]
```diff
@@ -39,6 +41,16 @@
...
+
+1. [Image_streamer_artifact_bundle](#image_streamer_artifact_bundle)
+2. [Image_streamer_build_plan](#image_streamer_build_plan)
+3. [Image_streamer_deployment_group](#image_streamer_deployment_group)
+4. [Image_streamer_deployment_plan](#image_streamer_deployment_plan)
+5. [Image_streamer_golden_image](#image_streamer_golden_image)
+6. [Image_streamer_os_volume](#image_streamer_os_volume)
+7. [Image_streamer_plan_script](#image_streamer_plan_script)
+
 #### oneview_connection_template
```
Comment (fgbulsoni) [inline RESOURCES.md]: Missing a Title before the resources such as `## HPE OneView resources`, which should be linked by the summary item

File: RESOURCES.md [hunk 3]
```diff
@@ -39,6 +41,16 @@
 37. [Oneview_volume_attachment](#oneview_volume_attachment)
 38. [Oneview_volume_template](#oneview_volume_template)
 
+## HPE Image Streamer resources
```
Comment (fgbulsoni) [inline RESOURCES.md:44]: This should also be anchored to the corresponding section

**Review by fgbulsoni — CHANGES_REQUESTED** [review]
Missing a couple of anchors and titles

File: RESOURCES.md (hunk 1 above)
Comment (marikrg) [reply RESOURCES.md:3]: I'm not able to add a link for a heading with the same name.
I can link it with the first resource, oneview_connection_template
Is it okay for you?

File: RESOURCES.md (hunk 3 above)
Comment (marikrg) [reply RESOURCES.md:44]: Same comment as above.

File: RESOURCES.md (hunk 1 above)
Comment (marikrg) [reply RESOURCES.md:3]: Aesthetically, isn't good to have a link in a heading.
Especially in this case, since it'll not be linked to any specific content.

**Review by fgbulsoni — APPROVED** [review]

**Pull request merged**